packages/engine/architecture.md

```diff
@@ -0,0 +1,18 @@
+# The Project Layout
+
+Currently the hEngine consists of two binaries located within the [`./bin`](bin) folder.
+To read the documentation for the various components, run:
+
+```sh
+cargo doc --open --workspace --no-deps
```

Comment:
Let's put the `--open` at the end, then it's easier to remove it after copy/pasting or re-run a command.
```suggestion
cargo doc --workspace --no-deps --open 
```